Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -152,15 +154,19 @@ public class PushDeploymentController : ApiController
             using (_tracer.Step("OnePushDeploy"))
             {
                 var response = Request.CreateResponse();
+
                 //Parses all query parameters into queryCollection
                 var queryCollection = Request.RequestUri.ParseQueryString();
                 var deployType = queryCollection[Constants.Type];
-                var website_framework = System.Environment.GetEnvironmentVariable(Constants.Framework).ToLower();
+                //var website_framework = System.Environment.GetEnvironmentVariable(Constants.Framework).ToLower();
+                var website_framework = "tomcat";
```

Comment:
I am aware that this should not be here, it was left over from testing. It will be removed for the next revision